anchor/client/src/cli.rs

```diff
@@ -30,6 +31,83 @@ pub static LONG_VERSION: LazyLock<String> = LazyLock::new(|| {
 
 pub const FLAG_HEADER: &str = "Flags";
 
+#[derive(Parser, Debug, Clone, Deserialize, Serialize)]
+pub struct LoggingFlags {
```

Comment:
This is more of a stylistic thing, and it will depend on how the rest of the anchor project is structured, but personally I'd try to keep all logging code inside the `logging` crate.

Then only import `LoggingFlags` (either here in `Node` or in `main.rs` inside `Cli`: see my other comment) combined with `#[clap(flatten)]` would make it all very tidy imo